.github/scripts/replace_pip_install_notebooks.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Check if an argument was provided
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 path_to_notebook"
+    exit 1
+fi
+
+# Assign the first argument to the variable notebook
+notebook=$1
+
+# Now you can use the variable $notebook in your script
+echo "The path to the notebook is \"$notebook\""
```

Comment:
Perhaps some (most?) of those echo should use [the debug output option/prefix](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#setting-a-debug-message), so that we only show what is necessary on a normal run?

Reply:
Yes, didn't know about this. Thx! 
Will fix the next times these files are touched!